content/tools/oddcontrast.md

```diff
@@ -0,0 +1,60 @@
+---
+title: OddContrast
+sub: Color contrast checker with Oklch, Oklab, P3, and more
+feature: true
+date: 2022-12-31
+logo: oddcontrast
+image:
+  position: top
+  src: projects/contrast.jpg
+oss: owner
+author: oddbird
+index: OddContrast
+links:
+  site: https://www.oddcontrast.com/
+  source: https://github.com/oddbird/oddcontrast
+summary: |
+  OddContrast is a color space converter,
+  featuring newer color spaces like
+  Oklch, Oklab, and P3.
+  It's also a color contrast checker
+  to help web designers meet
+  WCAG 2 accessibility standards.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+## Support for New Color Spaces
+
+In additional to Oklch, Oklab, and P3, OddContrast also includes hex, HSL, lab,
+LCH, and sRGB. Enter a color from any color space and convert the color to any
+other space. OddContrast provides an alert if the color you entered is out of
+gamut for a given color space. Adjust the sliders until your desired color space
+is back in gamut.
+
+{{ embed.img(
+  src='projects/out-of-gamut.jpg',
+  alt='display-p3 light yellow color showing hexidecimal color
+    outside the sRGB gamut and the hsl color outside the hsl gamut'
+) }}
+
+## Contrast Checker for Accessibility
+
+OddContrast checks the contrast ratio between two colors and displays pass or
+fail information based on the WCAG 2 accessibility standards for AA and AAA in
+normal and large font sizes. If you need a reminder of contrast ratio standard
```

Comment:
```suggestion
normal and large font sizes. If you need a reminder of the contrast ratio requirements
```

Reply:
@SondraE wanted to double check with you on this one... I don't mind if you don't take the suggestions, but in the phrase "If you need a reminder of contrast ratio standard"  shouldn't that last word either be standards or the phrase would need a "the" before the word contrast? Or am I reading this out of context?